flow-typed/box-ui-elements.js

```diff
@@ -696,6 +696,15 @@ type Integration = {
     type: APP_INTEGRATION,
 };
 
+type AdditionalTab = {
+    callback: typeof Function,
+    element: React.Element<any>,
+    id: number,
+    title: ?string,
```

Comment:
status?

Reply:
I realized status (or any other prop) they care about that's not needed for rendering doesn't need to be typed by us. We'll call their callback with whatever props they give us, but the sidebar doesn't actually care about what they are.